frameworks/bf/run-bf.js

```diff
@@ -0,0 +1,7 @@
+var execSync = require('child_process').execSync;
+module.exports.runBf = function(input) {
```

Comment:
If you have `module.exports.runBf`, `require('/runner/frameworks/bf/run-bf.js');` will be `{runBf}`.

Try doing `module.exports = function runBF(input)` instead.

You can also do `const runBF = require('/runner/frameworks/bf/run-bf.js').runBf;` but I think it's unnatural.

Another option is just embed it directly after

```js
require('/runner/frameworks/javascript/cw-2');
```

which might make more sense because it's just a single function.